packages/sanity/src/structure/comments/plugin/inspector/CommentsInspector.tsx

```diff
@@ -119,18 +121,18 @@ function CommentsInspectorInner(props: DocumentInspectorProps) {
           pushToast({
             closable: true,
             status: 'info',
-            title: 'Copied link to clipboard',
+            title: t('comments.inspector-copy-title'),
```

Comment:
Should we be more specific here and name the keys to e.g:

- `comments.copy-link-success-message`
- `comments.copy-link-error-message`